src/http/actions/actions.customers.ts

```diff
@@ -44,21 +44,22 @@ export module CustomerActions {
    */
   export const findAll = (callback: Function) => {
     const queryString = `SELECT * FROM ${_tableName}`;
-    db.query(queryString, (err, result) => {
-      if (err) {
-        callback(err);
+    _runner.run(queryString, (res: SQLQueryResponse<Array<TinyCustomer>>)=>{
```

Comment:
I found that I need to define a type for responses in actions if I whant to use res.map() for parse directions JSON, but when I do it can't parse JSON becouse item.direction is a Direction not a string. How must I solve it?.

Reply:
I'v tried to use SQLQueryResponse<Array<Object>> instead, but then can't use item properties in map becouse "Object type has not bla,bla property"

Reply:
Let's going to make a call